storyruntime/App.py

```
@@ -96,6 +100,29 @@ def image_pull_policy(self):
         await self.expose_services()
         await self.run_stories()
 
+    def create_tmp_dir(self):
+        if self._tmp_dir_created:
+            return
+
+        self._tmp_dir_created = True
+
+        path = self.get_tmp_dir()
+        self.logger.debug(f'Creating tmp dir {path} (on-demand)')
+        try:
+            pathlib.Path(path).mkdir(parents=True, mode=0o700, exist_ok=True)
+        except BaseException as e:
+            self.logger.error(f'Failed to create tmp dir {path}', e)
```

Shouldn't we let the app crash if this fails?

---

I went ahead and updated it to re-throw the exception, but I'm not entirely sure how we would test something without a basic unit test.